docs/CONTRIBUTING.md

```diff
@@ -19,10 +19,13 @@ Welcome! This guide covers everything you need to know to contribute components
 Before contributing, ensure you have the following tools installed:
 
 - **Python 3.11+** for component development
-- **uv** ([installation guide](https://docs.astral.sh/uv/getting-started/installation)) to manage Python dependencies including `kfp` and `kfp-kubernetes` packages
-- **pre-commit** ([installation guide](https://pre-commit.com/#installation)) for automated code quality checks
+- **uv** ([installation guide](https://docs.astral.sh/uv/getting-started/installation)) to manage
```

Comment:
Please reformat these markdown files now that you added the line length of 120 in the markdown lint config. It seems this is enforcing 100 character length lines.